mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt

```diff
@@ -16,6 +18,7 @@ import com.mifos.utils.*
 /**
  * Created by mayankjindal on 22/07/17.
  */
+@Suppress("DEPRECATION")
```

Comment:
Is this required? as you marked the whole file for deprecation.

Reply:
I have removed this.